src/components/dialogs/notificationDialogUtils/NotificationIndividualMergeRequestDialog.jsx

```diff
@@ -11,58 +12,50 @@ export default function NotificationIndividualMergeDialog({
   notification,
 }) {
   const notificationType = notification?.message_type;
+  const mergeRequestId = notification?.message_values?.request_id;
   const currentNotificationSchema = get(
     notificationSchema,
     notificationType,
   );
-  const path = get(currentNotificationSchema, 'path');
   const {
-    mutate: patchCollaboration,
-    error,
-    isError,
-  } = usePatchCollaboration(); // TODO once individual merge is fleshed out, change this
-  // TODO will likely need individual IDs
+    mutate: blockMerge,
+    error: blockError,
+    loading: blockLoading,
+  } = useBlockMerge();
 
-  const onClickGrant = async () => {
-    const temp = 'changeMe';
-    const response = await patchCollaboration({
-      collaborationGuid: temp,
-      operations: [
-        // TODO this will change
-        {
-          op: 'replace',
-          path,
-          value: 'approved',
-        },
-      ],
+  const {
+    mutate: allowMerge,
+    error: allowError,
+    loading: allowLoading,
+  } = useAllowMerge();
+
+  const error = blockError || allowError;
+  const isError = blockError || allowError;
```

Comment:
`error` and `isError` have the same value.

Reply:
Whoops

Reply:
Done